src/main/java/com/google/sps/utils/data/PopulationTrend.java

```diff
@@ -0,0 +1,8 @@
+package com.google.sps.utils.data;
+
+public enum PopulationTrend {
+    INCREASING,
+    DECREASING,
+    STEADY,
+    UNKNOWN
```

Comment:
Nice. Hopefully we'll get to displaying the trend as a timeline/graph, but if not, this might be unnecessary. Or we could always just have a text entry.